roles/lib_openshift/library/oadm_manage_node.py

```diff
@@ -748,6 +748,29 @@ class OpenShiftCLIError(Exception):
     pass
 
 
+def locate_oc_binary():
```

Comment:
I'd prefer it if this moved to a static method under the Utils class.  Seems like the perfect place for it.